index.html

```diff
@@ -0,0 +1,134 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="utf-8" />
+  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+  <meta name="viewport" content="width=device-width, initial-scale=1" />
+  <title>Personal portfolio</title>
+
+  <link rel="Stylesheet" href="./styles.css" />
+</head>
+
+
+<body>
+  <header class="main-header">
+    <a href="#" class="logo">Welcome</a>
+    <nav class="nav">
+      <ul class="nav-menu">
+        <li class="nav-link"><a href="#Portfolio">Portoflio</a></li>
+        <li class="nav-link active"><a href="#About">About</a></li>
+        <li class="nav-link"><a href="#form">Contact</a></li>
+      </ul>
+
+    </nav>
+  </header>
+
+  <main class="container">
+    <section class="main-section">
```

Comment:
- [ ] Please use the background image of the template you choose. You can't mix the content of various templates. I think you are doing template 3, but this background is from template 1. Please add the correct background image to the section 👍